src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -98,6 +105,16 @@
     (when (or differing-svc-version? differing-status-version?)
       (throw (IllegalStateException. error-message)))))
 
+(defn validate-protocol
```

Comment:
We typically name functions like this with a `!` at the end to indicate that they don't return a meaningful value and are only called for their side effects - i.e. `validate-protocol!`.